lib/api/registry/reg.js

```diff
@@ -57,10 +57,13 @@ function regCmd(args) {
 
         proc.on('close', code => {
             if (code !== 0) {
-                reject(`Error when calling reg.exe: ${buffer}`);
-            } else {
-                resolve(buffer);
+                // The reg.exe command exits with code 1 if a query returns
+                // zero results. However, this could also indicate that there
+                // is an error, so logging it for traceability reasons.
+                console.log(`The reg.exe command exited with code ${code}. ` +
```

Comment:
Will this log be visible in production release of Core?

Reply:
Yes, it will be visible on the console when pressing Ctrl+Shift+I.